app/domain/authentication/authn_gcp/consts.rb

```diff
@@ -7,6 +7,8 @@ module AuthnGcp
     INSTANCE_NAME_RESTRICTION_NAME = "instance-name"
     SERVICE_ACCOUNT_ID_RESTRICTION_NAME = "service-account-id"
     SERVICE_ACCOUNT_EMAIL_RESTRICTION_NAME = "service-account-email"
+    PERMITTED_CONSTRAINTS = %w(authn-gcp/instance-name authn-gcp/project-id authn-gcp/service-account-id authn-gcp/service-account-email).freeze
```

Comment:
you can add `# frozen_string_literal: true` to the head of the file and remove the `.freeze` here